scripts/upgrade.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers, upgrades } from 'hardhat';
+import { env } from '../config/env';
+import { getDeploymentConfig } from '../deploy/deploy';
+import { VoucherHub } from '../typechain-types';
+import { upgradeProxy } from './voucherHubUtils';
+
+async function upgradeVoucherHub() {
+    console.log(`Upgrading VoucherHub contract ...`);
+
+    const chainId = (await ethers.provider.getNetwork()).chainId.toString();
+    console.log('ChainId:', chainId);
+
+    const config = await getDeploymentConfig(Number(chainId));
+    if (!config.voucherHubAddress && !env.IEXEC_VOUCHER_HUB_ADDRESS) {
+        throw new Error(`No VoucherHub deployed on the target chain ${chainId}`);
+    }
+
+    const voucherHubProxyAddress = (config.voucherHubAddress || env.IEXEC_VOUCHER_HUB_ADDRESS)!;
+
+    // Fetch proxy admin details
+    const VoucherHubFactoryUpgrade = await ethers.getContractFactory('VoucherHub');
```

Comment:
```suggestion
    const voucherHubFactoryUpgrade = await ethers.getContractFactory('VoucherHub');
```
We try to use camelCase for variables.

Reply:
fix Commit: daec0d874a32409a0142522c0b072fc05d350182